_posts/2015-12-16-API-Blueprint-Future.md

```diff
@@ -0,0 +1,100 @@
+---
+title: This is going to be Awesome
+excerpt: On the future of API Blueprint
+layout: post
+date: 2015-12-16 00:00:00 +0100
+author: zdenek
+published: true
+comments: true
+---
+
+What would be API Blueprint like if it started today? Different vocabulary, features or grammar concepts?
+
+The truth is, in past three years, API Blueprint was putting its flag in the sand. Carving out its concepts and defining the emerging API design space.
+
+From its beginnings API Blueprint stood out for its simplicity and collaborative nature. The aim has always been to create a simple, concise yet powerful language. A language in which everybody can think about the design of an API.
```

Comment:
"From its beginning, API Blueprint..."

"concise yet powerful language—a language in which everybody can think about the design of an API."